gateway/configuration.go

```diff
@@ -43,6 +43,10 @@ func loadConfig() {
 	config.RegisterDurationConfigVariable(time.Duration(10), &WriteTimeout, false, time.Second, []string{"WriteTimeout", "WriteTimeOutInSec"}...)
 	config.RegisterDurationConfigVariable(time.Duration(720), &IdleTimeout, false, time.Second, []string{"IdleTimeout", "IdleTimeoutInSec"}...)
 	config.RegisterIntConfigVariable(524288, &MaxHeaderBytes, false, 1, "MaxHeaderBytes")
+	//default is set with assumption that 4GB of RAM is present, with avg. event size=5KB & avg. batch size=10.
+	config.RegisterIntConfigVariable(80000, &maxClients, false, 1, "Gateway.maxClients")
+	config.RegisterBoolConfigVariable(true, &enableClientLimit, false, "Gateway.enableClientLimit")
```

Comment:
I actually wanted to have a default values, so that anyone how is enabling doesn't have to think & calculate, what the appropriate number should be. And, can use the default one. But, since we don't have much clarity on what that number should be, I will remove two variable and combine them into one. As you mentioned.